csl-citation.json

```diff
@@ -1,101 +1,103 @@
 {
+    "$id": "https://raw.githubusercontent.com/citation-style-language/schema/master/csl-citation.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
     "description": "JSON schema for CSL citation objects",
-    "$schema" : "http://json-schema.org/draft-03/schema#",
-    "id": "https://github.com/citation-style-language/schema/raw/master/csl-citation.json",
+    "definitions": {
+        "citationItemDefinition": {
+            "type": "object",
+            "properties": {
+                "id": {
+                    "type": [
+                        "string",
+                        "number"
+                    ]
+                },
```

Comment:
The original schema requires "id" for each citation item, which is a requirement we should keep.